.github/workflows/ci.yaml

```diff
@@ -66,7 +66,7 @@ jobs:
 
       - name: Test
         run: |
-          ./project/scripts/sbt ";compile ;test"
+          ./project/scripts/sbt ";dist/pack; compile ;test"
```

Comment:
This re-introduces bootstrapping the compiler in "test_non_bootstrapped", which is annoying (can't see the results of running the tests in CI while the compiler fails to bootstrap).

If these tests need a bootstrapped compiler, can't they be in bootstrap-only?

Reply:
> If these tests need a bootstrapped compiler, can't they be in bootstrap-only?

I don't know enough about the test architecture to offer a reply, perhaps this is a question for @BarkingBad ?